sdk/turing/ensembler.py

```diff
@@ -279,6 +280,31 @@ def update(
             with the model. This will be passed to turing.ensembler.PyFunc.initialize().
             Example: {"config" : "config/staging.yaml"}
         """
+
+        # First we need to check if there are active job / router using the ensembler
+        # This check is done in the SDK to prevent the creation of new mlflow run
+
+        # check any active router version
+        relatedRouterVer = turing.Router.list_router_versions_with_filter(
+            ensembler_id=self._id, status=[RouterStatus.PENDING], is_current=False
+        )
+        if len(relatedRouterVer) > 0:
+            # if there is any active router version, the deletion process are restricted
+            raise ValueError("There is pending router version using this ensembler")
+
+        # check any active ensembling jobs
+        relatedJob = EnsemblingJob.list(
+            ensembler_id=self._id,
+            status=[
+                EnsemblingJobStatus.BUILDING,
+                EnsemblingJobStatus.PENDING,
+                EnsemblingJobStatus.RUNNING,
+            ],
+        )
+        if len(relatedJob) > 0:
+            # if there is any active ensembling jobs, the deletion process are restricted
+            raise ValueError("There is pending ensembling job using this ensembler")
```

Comment:
Same comment as for the routers - will be useful to have some instruction to the user on what needs to be done to overcome this error.